src/Core/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ project(ObEngineCore)
 
 include(group_files)
 
+add_custom_target(ConfigureObEngineGit COMMAND ${CMAKE_COMMAND} 
```

Comment:
Why configure the file as a custom target rather than during the CMake Configure ?

Reply:
See https://github.com/Sygmei/ObEngine/pull/319/files/07e3b92a527d79b1477f80b914d129f1688dfc8e#r499263010 for the reason.